### ISSUE TITLE
A narrow gap occurs between code map and scrollbar

### ISSUE THREAD

**Issue by JieJiSS** [Microsoft/vscode#42183]
- VSCode Version: 1.19.3
- OS Version: Windows 10 x64 1511

This seems to be a layout bug.

Steps to Reproduce:

1. Find a 1920*1080 screen
2. Open VSCode
3. Use `win+leftArrow` to move VSCode to the left part of the screen

You can see a narrow gap between the code map and the scrollbar.

![scr 18-01-26 13-16-33](https://user-images.githubusercontent.com/24671280/35426208-2bef18bc-029c-11e8-9558-354e4c515231.jpg)

![bandicam-2018-01-26-13-26-15-168](https://user-images.githubusercontent.com/24671280/35426359-5c8d7c2e-029d-11e8-9077-d37e2c3b4d04.gif)

Does this issue occur when all extensions are disabled?: Yes

I disabled all my extensions and still, it occurs:

![image](https://user-images.githubusercontent.com/24671280/35426477-61026f5c-029e-11e8-8a81-924e9b223550.png)

**Comment by JieJiSS** [comment]
And if I change the width of the VSCode editor, this bug might disappear.
Also, this bug occurs when `window_width=880` (using Snipaste to find out)
![image](https://user-images.githubusercontent.com/24671280/35426667-a8dede2c-029f-11e8-8546-38ca0e205e12.png)

**Comment by JieJiSS** [comment]
Well, actually my OS version is Windows 10 x64 1511 10586.104

**Comment by JieJiSS** [comment]
![scr 18-01-29 21-49-54](https://user-images.githubusercontent.com/24671280/35513591-93b747a4-053e-11e8-999c-97ddd11289fa.jpg)

updated my os to Windows 10 x64 1709 16299.125, the same error still occurs.

**Comment by alexandrudima** [comment]
@JieJiSS 

I've seen this issue before -- #34974 , but I could never reproduce.

* Are you using some form of `zoomLevel` in your settings ?

* I've pushed a small tweak in how the minimap is positioned recently. This tweak is in the Insiders Release. Can you please try to see if the insiders version still produces this -- https://code.visualstudio.com/insiders/

**Comment by JieJiSS** [comment]
@alexandrudima You're right. I found `window.zoomLevel: 1` in my settings.json, and after I removed it, I cannot reproduce this bug either.
I'm sorry to say that now I'm in China(UTC+0800) and it is 11:58 here so maybe I'll have to try the insider version tomorrow.
thanks very much!

**Comment by JieJiSS** [comment]
@alexandrudima Tested on insider version.

Ummm, the patch *sometimes* works.
If I resize the window by dragging the edge slowly, sometimes I can see the narrow gap. But if I resize the window with `Win+←`, then everything's right.
What's more, if I manage to produce a gap by dragging the edge (after I have moved the window to the left of the screen with `Win+←`) and I press the `Win+↑` to move it to the upper-left, then the gap disappear (even if I didn't change the width of the window).

btw, I found another 1px gap this time (at the bottom of the code map):

![scr 18-01-30 22-38-22](https://user-images.githubusercontent.com/24671280/35572017-a21da5e2-060e-11e8-8a42-a7895fce0410.jpg)
